README.md

```diff
@@ -27,11 +25,18 @@ Active Shipping is currently being used and improved in a production environment
 
 ## Installation
 
+Using bundler, add to the `Gemfile`:
+
 ```ruby
-gem install active_shipping
+gem 'active_shipping'
+```
+
+Or stand alone:
+
+```
+$ gem install active_shipping
 ```
 
-...or add it to your project's [Gemfile](http://bundler.io/).
 
 ## Sample Usage
```

Comment:
I'll add it. But we may want to fix this or be more explicit/opinionated about it.